test/openAPI/features/support/wallet_generate_link_code.js

```diff
@@ -0,0 +1,116 @@
+const chai = require('chai');
+const { spec } = require('pactum');
+const { Given, When, Then, Before, After } = require('@cucumber/cucumber');
+const {
+  localhost,
+  contentTypeHeader,
+  defaultExpectedResponseTime,
+  walletGenerateLinkCodeEndpoint,
+  X_XSRF_TOKEN,
+  transactionId,
+  walletGenerateLinkCodeResponseSchema,
+} = require('./helpers/helpers');
+
+chai.use(require('chai-json-schema'));
+
+let specWalletGenerateLinkCode;
+
+const baseUrl = localhost + walletGenerateLinkCodeEndpoint;
+const endpointTag = { tags: `@endpoint=/${walletGenerateLinkCodeEndpoint}` };
+
+Before(endpointTag, () => {
+  specWalletGenerateLinkCode = spec();
+});
+
+// Scenario: Successfully generate link code smoke type test
+Given('Wants to generate link code', () => 'Wants to generate link code');
+
+When(
+  'Send POST \\/linked-authorization\\/link-code request with given X-XSRF-TOKEN header, transactionId and {string} as requestTime',
+  (requestTime) =>
+    specWalletGenerateLinkCode
+      .post(baseUrl)
+      .withHeaders(X_XSRF_TOKEN.key, X_XSRF_TOKEN.value)
+      .withJson({
+        requestTime: requestTime,
+        request: {
+          transactionId: transactionId,
+        },
+      })
+);
+
+Then(
+  'Receive a response from the \\/linked-authorization\\/link-code endpoint',
+  async () => await specWalletGenerateLinkCode.toss()
+);
+
+Then(
+  'The \\/linked-authorization\\/link-code endpoint response should be returned in a timely manner 15000ms',
+  () =>
+    specWalletGenerateLinkCode
+      .response()
+      .to.have.responseTimeLessThan(defaultExpectedResponseTime)
+);
+
+Then(
+  'The \\/linked-authorization\\/link-code endpoint response should have status 200',
+  () => specWalletGenerateLinkCode.response().to.have.status(200)
+);
+
+Then(
+  'The \\/linked-authorization\\/link-code endpoint response should have content-type: application\\/json header',
+  () =>
+    specWalletGenerateLinkCode
+      .response()
+      .should.have.header(contentTypeHeader.key, contentTypeHeader.value)
+);
+
+Then(
+  'The \\/linked-authorization\\/link-code endpoint response should match json schema',
+  () => {
+    walletGenerateLinkCodeResponseSchema.properties.errors = [];
```

Comment:
@KarolinaKopacz Are you sure we need this line? Errors ain't required in the schema.

Reply:
Yes, we need it because there should be no errors so array should be empty

Reply:
so should we create a test step that will check if the error array is empty? The schema does not check it atm. I believe it only describes that `errors` property of the response should be an array (if response has the property)

Reply:
This step check it. This is why first I change the `walletGenerateLinkCodeResponseSchema.properties.errors` to empty array and the check if response schema is equals to `walletGenerateLinkCodeResponseSchema`

Reply:
But I will check it again

Reply:
It does not check if the errors array is empty. Try to put 
```
{
      "errorCode": "invalid_transaction_id",
      "errorMessage": "string"
}
```

in the mockoon in the positive response (in errors array) and you will see that all tests succeed and the response still matches with the schema, even if errors array is not empty.

Reply:
I changed the code a little bit